### PR TITLE
Settings/Contact Us 

### PR DESCRIPTION
**Pull request by anastasiaShpakava**
## Summary

Added Contac Us to Settings toolbar

## Reasons
User will  be able to contact the app support team conveniently from the mobile app settings. This will allow user to easily reach out to the support team for any questions, feedback, or issues related to the app.

## References

closes https://github.com/SoftTeco/AndroidAppTemplate/pull/112

### CODE REVIEW

File: app/src/main/kotlin/com/softteco/template/ui/feature/settings/SettingsScreen.kt [hunk 1]
```diff
@@ -100,6 +101,19 @@ private fun ScreenContent(
                 },
             )
             Divider()
+            ListItem(
+                modifier = Modifier.clickable {
+                    context.sendMail(recipient = "softteco.os.dev@gmail.com", subject = "User Inquiry or Feedback")
```
Comment (bstolyarov) [inline app/src/main/kotlin/com/softteco/template/ui/feature/settings/SettingsScreen.kt:106]: Let's move "softteco.os.dev@gmail.com" and "User Inquiry or Feedback" to constants.

File: app/src/main/kotlin/com/softteco/template/ui/feature/settings/SettingsScreen.kt [hunk 2]
```diff
@@ -100,6 +101,19 @@ private fun ScreenContent(
...
+                },
+                headlineContent = { Text(text = stringResource(id = R.string.contact_us)) },
+                trailingContent = {
+                    Icon(
+                        Icons.Sharp.ArrowForwardIos,
+                        contentDescription = "Settings description"
```
Comment (bstolyarov) [inline app/src/main/kotlin/com/softteco/template/ui/feature/settings/SettingsScreen.kt:112]: We translated a contentDescription in this app before. Let's use string res for this as well. Also, it should be different names for contentDescription for different list items.

File: app/src/main/kotlin/com/softteco/template/ui/feature/settings/SettingsScreen.kt [hunk 3]
```diff
@@ -100,6 +101,19 @@ private fun ScreenContent(
                 },
             )
             Divider()
+            ListItem(
```
Comment (bstolyarov) [inline app/src/main/kotlin/com/softteco/template/ui/feature/settings/SettingsScreen.kt:104]: Could you make a common list item component for this list

File: app/src/main/kotlin/com/softteco/template/utils/SendEmailExtension.kt
Comment (bstolyarov) [inline app/src/main/kotlin/com/softteco/template/utils/SendEmailExtension.kt:1]: Let's name this file `ContextExtension`

**Review by bstolyarov — CHANGES_REQUESTED** [review]

**Review by bstolyarov — APPROVED** [review]

**Review by EvgeniyFilippov — APPROVED** [review]

**Pull request merged**